gradio/components/annotated_image.py

```diff
@@ -15,8 +15,6 @@
 
 set_documentation_group("component")
 
-PIL.Image.init()  # fixes https://github.com/gradio-app/gradio/issues/2843
```

Comment:
We should keep this change as older versions of PIL (before 9.4.0) have the reported bug. Gradio pins to PIL 8.0 or higher